pyccel/codegen/codegen.py

```diff
@@ -233,7 +233,7 @@ def doprint(self, **settings):
         language = settings.pop('language', 'fortran')
 
         if not language in ['fortran', 'c', 'python']:
-            raise ValueError('the language {} not available'.format(lanugage))
+            raise ValueError('the language {} not available'.format(language))
```

Comment:
Hi @rakati thanks for this PR!

Since we are fixing a typo here, why not also improving the error message to something that is correct English? I suggest the following:
```
raise ValueError('{} language is not available'.format(language))
```

Reply:
done